cluster/gce/manifests/pdcsi-controller.yaml

```diff
@@ -0,0 +1,165 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: pdcsi-controller
+  namespace: kube-system
+  labels:
+    app: gcp-compute-persistent-disk-csi-driver
+    component: pdcsi-controller
+spec:
+  securityContext:
+    seccompProfile:
+      type: RuntimeDefault
+    runAsUser: {{runAsUser}}
+    runAsGroup: {{runAsGroup}}
+  containers:
+  - args:
+    - --v=5
+    - --csi-address=/csi/csi.sock
+    - --feature-gates=Topology=true
+    - --http-endpoint=:22011
+    - --leader-election-namespace=$(PDCSI_NAMESPACE)
+    - --timeout=250s
+    - --extra-create-metadata
+    - --leader-election
+    - --default-fstype=ext4
+    - --kubeconfig=/etc/srv/kubernetes/pdcsi-controller/kubeconfig
+    env:
+    - name: PDCSI_NAMESPACE
+      valueFrom:
+        fieldRef:
+          fieldPath: metadata.namespace
+    image: k8s.gcr.io/sig-storage/csi-provisioner:v2.2.1
+    livenessProbe:
+      failureThreshold: 1
+      httpGet:
+        path: /healthz/leader-election
```

Comment:
Why are specifying the leader-election sub resource for the liveness probe? Would it make more sense to just hit a /healthz endpoint and then let it delegate to leader-election and whatever else was appropriate?

Reply:
@verult Do you know the story here?

Reply:
This is the only health check that exists today in the CSI sidecars, so it's probably overkill to add a delegation mechanism. Without such mechanism I think it's useful to be explicit about exactly which health check the probe is hitting.

Reply:
Does this answer your question, Walter?